packages/wrapper-react/src/index.tsx

```diff
@@ -30,72 +29,49 @@ export const MonacoEditorReactComp: React.FC<MonacoEditorProps> = (props) => {
     const containerRef = useRef<HTMLDivElement>(null);
 
     useEffect(() => {
```

Comment:
We need a second `useEffect` for destruction, otherwise the editor resources are not properly destroyed:
```js
useEffect(() => {
  return () => {
    destroyMonaco();
  };
}, []);
```

Reply:
> Also, moving all methods outside the main useEffect will require wrapping them inside a useCallback which I think we can avoid by just including the methods inside the main useEffect.

Yes, thank you, that is the cleaner solution and I just verified that it works. It would be like that with only one `useEffect` 
```ts
    useEffect(() => {
       ...
        (async () => {
            await initMonaco();
            await startMonaco();
        })();
        return () => { destroyMonaco(); };
    }, [wrapperConfig]);
```